starrocks/benchmark.sh

```diff
@@ -2,18 +2,19 @@
 
 # This benchmark should run on Amazon Linux
 
-VERSION=3.0.0-preview
-DOWNLOAD_URL=https://releases.starrocks.io/starrocks/StarRocks-3.0.0-preview.tar.gz
+set -e
+
+VERSION=3.4.2-ubuntu-amd64
```

Comment:
I guess we can make our lifes easier and use the Starrocks docker image, see https://github.com/ClickHouse/JSONBench/tree/main/starrocks (install.sh / uninstall.sh)

Reply:
Agree, I thought about it, but decided to not make "radical" changes at once.